bindings/python/pymanifold.cpp

```diff
@@ -624,12 +574,20 @@ PYBIND11_MODULE(pymanifold, m) {
       "[Clipper2](http://www.angusj.com/clipper2/Docs/Overview.htm) library "
       "for polygon clipping (boolean) and offsetting operations.")
       .def(py::init<>())
-      .def(py::init(
-               [](PolygonsWrapper &contours, CrossSection::FillRule fillrule) {
-                 return CrossSection(*contours.polygons, fillrule);
-               }),
-           py::arg("contours"),
-           py::arg("fillrule") = CrossSection::FillRule::Positive)
+      .def(py::init([](std::vector<std::vector<Float2>> &polygons,
+                       CrossSection::FillRule fillrule) {
+             std::vector<SimplePolygon> simplePolygons(polygons.size());
+             for (int i = 0; i < polygons.size(); i++) {
+               std::vector<glm::vec2> vertices(polygons[i].size());
+               for (int j = 0; j < polygons[i].size(); j++) {
+                 vertices[j] = {std::get<0>(polygons[i][j]),
+                                std::get<1>(polygons[i][j])};
+               }
+               simplePolygons[i] = {vertices};
+             }
+             return CrossSection(simplePolygons, fillrule);
+           }),
```

Comment:
I think `FillRule` should be made a [default arg](https://pybind11.readthedocs.io/en/stable/advanced/functions.html#default-arguments-revisited) with `py::arg` as other `.def`s in `pymanifold.cpp` have done.